lib/bundling-job.js

```diff
@@ -6,10 +6,15 @@ import { createArchive } from './archive';
 
 async function runBundlingJob (job, archiveName = 'archive.zip') {
   const files = await findCollectionFileMembers(job.used);
+  // If we encounter a filename we've seen before, we append
+  // the file's ID
+  const fileNames = new Set();
   const archivingParams = files.map((f) => {
+    const duplicateName = fileNames.has(f.name);
+    fileNames.add(f.name);
     return {
       path: f.physicalUri.replace('share://', '/share/'),
-      name: sanitize(f.name, { replacement: '_' })
+      name: sanitize(`${f.name}${duplicateName ? ` ${f.id}` : ''}`, { replacement: '_' })
```

Comment:
don't think this is the expected result?
PDF files may be broken
![image](https://github.com/user-attachments/assets/acb8c6c6-265c-4a88-a5f5-36fbc5b4e394)

Reply:
Also I get the limitation with not wanting to query the document container position.
UUID in filename doesn't look very appealing though.
adding (1) maybe but if we do that without actually getting the right numbers it will be confusing if doc 4 gets (1), doc 3 gets (2) etc..